catalyst_voices/lib/widgets/common/label_decorator.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:catalyst_voices_brands/catalyst_voices_brands.dart';
+import 'package:collection/collection.dart';
+import 'package:flutter/material.dart';
+
+class LabelDecorator extends StatelessWidget {
```

Comment:
Should this be called VoicesLabelDecorator?

Reply:
Well i would say that `Voices` prefix makes sense when we do have some kind of collision of names like `Checkbox` and `VoicesChangebox`. When there is not similar widgets around i say lets keep names shorter